File: fontes/c/micro/micro11.c
#include<stdio.h>
int verifica(int n)
{
	int res;
	if (n > 0) 
	{
		res = 1;
	}
 	else if (n < 0)
	{
 	res = -1;
	}
	else
	{
	res = 0;
	}
	return res;
}

int main()
{
 int numero;
 int x;
 printf("Digite um número: ");
 scanf("%d", &numero);
 x = verifica(numero);

 if (x == 1)
 {
 	printf("Número positivo\n");
 }
 else if (x == 0)
 {
  printf("Zero\n");
 }
 else
 {
  	printf("Número negativo\n");
 }
 return 0;
}

File: fontes/pascal/micro/micro11.pas
program micro11;

 function verifica (n:integer) : integer;
 begin
 if n > 0 then
 verifica := 1
 else if n < 0 then
verifica := -1
 else
 verifica := 0;
 end;

 var
 numero,x : integer;

 begin
 write('Digite um numero: ');
 readln(numero);
 x := verifica(numero);
 if x = 1 then
 writeln('Numero positivo')
 else if x = 0 then
 writeln('zero')
else writeln('numero negativo');
 end.